DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/BasicAuthenticatedAccessibilityTests.cs

```diff
@@ -80,6 +80,7 @@ public class BasicAuthenticatedAccessibilityTests : AccessibilityTestsBase,
         [InlineData("/NotificationPreferences/Edit/DelegateUser", "Update notification preferences")]
         [InlineData("/ChangePassword", "Change password")]
         [InlineData("/TrackingSystem/Support", "Support")]
+        [InlineData("/TrackingSystem/Support/Tickets", "Support tickets")]

```

Comment:
I wouldn't be surprised if this would fail if there were an instance of the old site running to load the old site into the page. We don't have these tests for the other pages loading in the old site in iframes (Pricing, Find a Centre, LearningContent in the main area of the site).
I'm not sure if it is worth keeping this or not?

Reply:
Originally I had the old site running when I did this. I've since turned that off and re-tested. Under thos condition the test still ran fine.
But I know what you mean I could introducing a flaky test so I have dropped this.